setupTests.js

```diff
@@ -7,3 +7,12 @@ global.console = {
   info: console.info,
   debug: console.debug
 };
+
+// These are required in order to initialise the Airtable API
+// inside of `ResourcesGateway`. They need to be present before
+// it is required inside of dependencies.js.
+process.env = Object.assign(process.env, {
+  AIRTABLE_API_KEY: 'not.a.real.api.key',
+  AIRTABLE_BASE_ID: 'not.a.real.base.id',
+  AIRTABLE_TABLE_NAMES: 'not.a.real.table.name'
+});
```

Comment:
Not sure why these are required? You're not using dependencies.js in the tests?

Reply:
dependencies is imported in the endpoint file, and since it sets these up outside of any function etc. they're required immediately. Unfortunately (although probably a good thing really), the Airtable API throws if you try to construct it without an API key and so these are needed any time it is imported else it will throw.